site/docs/reference/working-logs-stats.md

```diff
@@ -83,3 +68,51 @@ acmeCo/anvils/logs:
         exclude:
           name: ['acmeCo/anvils/logs']
 ```
+
+## Available statistics
+
+Available statistics include information about the amount of data in inputs and outputs of each transaction. They also include temporal information about the transaction. Statistics vary by task type (capture, materialization, or derivation).
+
+A thorough knowledge of Flow's [advanced concepts](../concepts/README.md#advanced-concepts) is necessary to effectively leverage these statistics.
+
+`stats` collection documents include the following properties.
+
+### Shard information
+
+Each `stats` document begin with data about the shard processing the transaction.
```

Comment:
I think it would help to convey that the combination of `/shard/name`, `/shard/keyBegin`, and `/shard/rClockBegin` uniquely identifies a specific task processing shard.